apps/aecore/include/pow.hrl

```diff
@@ -1,2 +1,2 @@
 -define(HIGHEST_TARGET_SCI, 16#2100ffff).
--define(HIGHEST_TARGET_INT, 16#FFFFFF0000000000000000000000000000000000000000000000000000000000).
+-define(HIGHEST_TARGET_INT, 16#FFFF000000000000000000000000000000000000000000000000000000000000).
```

Comment:
can you please elaborate on the consequences of changing this difficulty?

Reply:
the reason for this change is that we cannot represent the previous value in scientific notation. The significand must be below 0x7fffff because it is a signed value so we lose accuracy. 
The consequence of it is minor (though it is stricter, unlike I said at standup): we drop about 1 - 0xffff00/0xffffff = 1.5x10^-5 of the hashes even with the maximum threshold.

Reply:
I see that this adapts the macro to the comment. Thanks for the link @akorosmezey